governance.md

```diff
@@ -0,0 +1,53 @@
+# Governance
+
+This contains information about some of the governance procedures that take place in the Node. Initially this covers the voting process
+
+## Create a Vote
+
+In order to create a vote, in the Polkadot JS Apps frontend select the Governance tab, then `Tech. Comm`. You will then be presented with a screen with the current Governance members.
+
+In order to start a new vote select proposals.
+
+<img width="1680" alt="Screenshot 2022-06-27 at 14 02 49" src="https://user-images.githubusercontent.com/35331926/175948593-aae8d69a-a722-47f8-8a58-a75c0d9af3d7.png">
+
+On the proposal screen select submit proposal on the right hand side. This will bring up a new proposal which must be made from an initial account, for example, here Alice is creating a proposal. You may notice the use of the doAs pallet, for more information visit [the readme](README.md) on how it works.
+
+<img width="1278" alt="3" src="https://user-images.githubusercontent.com/35331926/175949271-6d9b543a-95aa-4a49-b637-dd1668c179af.png">
+
+Once the proposal has been agreed and submit has been clicked another window will appear to confirm the transaction.
+
+<img width="1083" alt="4" src="https://user-images.githubusercontent.com/35331926/175949544-3c542242-46e5-4718-a242-babb19ef2fe2.png">
+
+Click sign and submit. There wiill then be a voting round.
+
+<img width="1666" alt="Screenshot 2022-06-27 at 14 13 34" src="https://user-images.githubusercontent.com/35331926/175950428-ac60c688-8448-4729-b770-bda0a0c082a2.png">
+
+<img width="720" alt="Screenshot 2022-06-27 at 14 14 08" src="https://user-images.githubusercontent.com/35331926/175950487-0765eb54-d8a0-4fe6-adbb-9dd62653b39a.png">
```

Comment:
Look at how we include images in other [repositories](https://raw.githubusercontent.com/digicatapult/wasp-documentation/main/docs/architecture.md)